File: src/main/java/frc/team4069/robot/commands/elevator/DebugCommand.java
package frc.team4069.robot.commands.elevator;

import frc.team4069.robot.commands.CommandBase;

public class DebugCommand extends CommandBase {
    public DebugCommand() {
        requires(elevator);
    }

    @Override
    protected void initialize() {
        System.out.println(elevator.getPosition());
    }

    @Override
    protected boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/team4069/robot/commands/elevator/DebugCommand.kt
package frc.team4069.robot.commands.elevator

import edu.wpi.first.wpilibj.command.InstantCommand
import frc.team4069.robot.subsystems.ElevatorSubsystem

class DebugCommand : InstantCommand() {
    init {
        requires(ElevatorSubsystem)
    }

    override fun initialize() = println(ElevatorSubsystem.position)
}